supplementary/alica_ros_turtlesim/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@ set(PKG_DEPS
     roscpp
```

Comment:
@mansiVerma26  did you compile alica_ros_turtlesime with changes? I expected more changes here.

Reply:
Yes I compiled it locally, and checked CI twice. It compiled successfully.